guides/main_concepts/relevancy.md

```diff
@@ -157,31 +157,18 @@ To add a rule to the existing ranking rule, you have to add the rule to the exis
 ]
 ```
 
-## Importance of the attributes
+## Attribute Ranking Order
 
-In a dataset, some fields are more relevant to the search than others. A `title`, for example, has a value more meaningful to a movie search than its `description` or its `director` name.
+In a typical dataset, some fields are more relevant to search than others. A `title`, for example, has a value more meaningful to a movie search than its `description` or its `release_date`.
 
-By default, the order of importance of the attributes is based on their order of appearance in the first document added. Then, each new attribute found in new documents will be added at the end of this ordered list.
+By default, the attribute ranking order is generated automatically based on the attributes' order of appearance in the indexed documents. However, it can also be set manually.
 
-If you wish to specify the order of the attributes you can either define them in the settings or set the correct order in the first document indexed.
-
-### Changing the order of the attributes
-
-You may want to change the order once the documents have been ingested. This is still very possible using the [searchable attributes list](/guides/advanced_guides/field_properties.md#searchable-fields).
-
-Whenever a document is added to MeiliSearch, all new attributes found in it are automatically added to two lists:
-
-- **The [searchable attributes list](/references/searchable_attributes.md)**: Attributes of the fields in which to search for matching query words.
-- **The [displayed attributes list](/references/displayed_attributes.md)**: Attributes of the fields displayed in documents.
-
-This searchable attributes list is **ordered**, which means the order in which the attributes appear in the list determines their relevancy. Attributes are arranged from the most important attribute to the least important attribute.
-
-Place the attributes in the desired order and send this updated list using the [settings routes](/references/settings.md). Attributes will be re-ordered.
+For a more detailed look at this subject, see our reference page for [the searchable attributes list](/guides/advanced_guides/field_properties.md#the-searchable-attributes-list).
 
 #### Example
 
 ```json
-["title", "description", "director"]
+["title", "description", "release_date"]
 ```
 
-If you take a look at the above order, the matching words found in `title` will make the document more relevant than one with the same matching words found in `description` or `director`.
+With the above attribute ranking order, matching words found in the `title` field would have a higher impact on relevancy than the same words found in `description` or `release_date`. If you searched "1984", for example, results like Michael Radford's film "1984" would be ranked higher than movies released in the year 1984.
```

Comment:
nice reference. The book is great, I forgot about the movie

Reply:
I've never seen it, but I thought it was a good example! Only problem is, it might confuse people who try to follow along with the movies.json dataset 🤔